### PR TITLE
Sasha/force fix

### PR DESCRIPTION
**Pull request by sasha-scale**
@tetranoir discovered a bug for local image upload.  In the process, we noticed that pytest tests were failing.  Digging into the problem, found some of the tests were malformed & found an issue with annotation_ids for segmentation masks.  We should think about maybe setting up a requirement that tests pass before merging to prevent this problem in the future.

### CODE REVIEW

File: tests/test_dataset.py [hunk 1]
```diff
@@ -145,6 +162,11 @@ def test_slice_append(dataset):
     slc.append(reference_ids=[item.reference_id for item in ds_items[:3]])
 
     response = slc.info()
+    print(response)
```
Comment (tetranoir) [inline tests/test_dataset.py:165]: Is this supposed to be here?

**Review by tetranoir — COMMENTED** [review]
Running pytest results in these failures
```
_______________________________________________ test_dataset_create_and_delete _______________________________________________

CLIENT = <nucleus.NucleusClient object at 0x111c8beb0>

    def test_dataset_create_and_delete(CLIENT):
        # Creation
        ds = CLIENT.create_dataset(TEST_DATASET_NAME)
        assert isinstance(ds, Dataset)
        assert ds.name == TEST_DATASET_NAME
        assert ds.model_runs == []
        assert ds.slices == []
        assert ds.size == 0
>       assert ds.items == []

tests/test_dataset.py:39:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
nucleus/dataset.py:53: in items
    return self._client.get_dataset_items(self.id)
nucleus/__init__.py:180: in get_dataset_items
    response = self._make_request(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

payload = {}, route = 'dataset/ds_c1k6bc57mwtg099jf2p0/datasetItems', requests_command = <function get at 0x110fef8b0>

    def _make_request_patch(
        payload: dict, route: str, requests_command=requests.post
    ) -> dict:
        response = client._make_request_raw(payload, route, requests_command)
>       assert response.status_code in SUCCESS_STATUS_CODES, \
            f"HTTP response had status code: {response.status_code}. " \
            f"Full JSON: {response.json()}"
E       AssertionError: HTTP response had status code: 500. Full JSON: {'status_code': 500, 'error': 'select "undefined"."id", "undefined"."metadata", "undefined"."ref_id", "undefined"."original_image_url" from "nucleus"."dataset_item" inner join "nucleus"."dataset" on "nucleus"."dataset"."id" = "nucleus"."dataset_item"."dataset_id" where "nucleus"."dataset"."id" = $1 and "nucleus"."dataset"."user_id" = $2 - missing FROM-clause entry for table "undefined"'}
E       assert 500 in [200, 201, 202]
E        +  where 500 = <Response [500]>.status_code

conftest.py:52: AssertionError
_________________________________________________ test_dataset_append_local __________________________________________________

CLIENT = <nucleus.NucleusClient object at 0x111c8beb0>, dataset = <nucleus.dataset.Dataset object at 0x111d26d30>

    def test_dataset_append_local(CLIENT, dataset):
        ds_items_local = [DatasetItem(image_location=LOCAL_FILENAME)]
>       response = dataset.append(ds_items_local)

tests/test_dataset.py:90:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
nucleus/dataset.py:155: in append
    return self._client.populate_dataset(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <nucleus.NucleusClient object at 0x111c8beb0>, dataset_id = 'ds_c1k6bcxpsjb00992t7ag'
dataset_items = [<nucleus.dataset_item.DatasetItem object at 0x111d264f0>], batch_size = 20, force = False

    def populate_dataset(
        self,
        dataset_id: str,
        dataset_items: List[DatasetItem],
        batch_size: int = 100,
        force: bool = False,
    ):
        """
        Appends images to a dataset with given dataset_id.
        Overwrites images on collision if forced.
        :param dataset_id: id of a dataset
        :param payload: { "items": List[DatasetItem], "force": bool }
        :param local: flag if images are stored locally
        :param batch_size: size of the batch for long payload
        :return:
        {
            "dataset_id: str,
            "new_items": int,
            "updated_items": int,
            "ignored_items": int,
            "upload_errors": int
        }
        """
        local_items = []
        remote_items = []

        # Check local files exist before sending requests
        for item in dataset_items:
            if item.local:
                if not item.local_file_exists():
>                   raise NotFoundError()
E                   nucleus.errors.NotFoundError: Could not open file. Check the path or if it exists.

nucleus/__init__.py:330: NotFoundError
================================================== short test summary info ===================================================
FAILED tests/test_dataset.py::test_dataset_create_and_delete - AssertionError: HTTP response had status code: 500. Full JSO...
FAILED tests/test_dataset.py::test_dataset_append_local - nucleus.errors.NotFoundError: Could not open file. Check the path...
========================================== 2 failed, 26 passed in 74.73s (0:01:14) ===========================================
```

File: tests/test_dataset.py (hunk 1 above)
Comment (sasha-scale) [reply tests/test_dataset.py:165]: good catch!

**Review by tetranoir — APPROVED** [review]
LGTM - tests pass after deleting tests/__init__.py

**Pull request merged**